### PR TITLE
Add class pointer-events-auto

### PR DESCRIPTION
**Pull request by PascalPiche**

### CODE REVIEW

**Comment by nitriques** [comment]
This now needs rebasing, sorry

File: workspace/assets/css/lib/class-pointer-events.less [hunk 1]
```diff
@@ -1,7 +1,9 @@
 #generate-core-pointer-events-classes (@suffix: ~'') {
 	.pointer-events-none@{suffix}                           {pointer-events: none;}
 	.pointer-events-all@{suffix}                            {pointer-events: all;}
+	.pointer-events-auto@{suffix}                           {pointer-events: auto;}
 
 	.pointer-events-none@{suffix}-when-selected.is-selected {pointer-events: none;}
 	.pointer-events-alld@{suffix}-when-selecte.is-selected  {pointer-events: all;}
-}
\ No newline at end of file
+	.pointer-events-auto@{suffix}-when-selecte.is-selected  {pointer-events: auto;}
```
Comment (nitriques) [inline workspace/assets/css/lib/class-pointer-events.less]: missing a `d`

File: workspace/assets/css/lib/class-pointer-events.less [hunk 2]
```diff
@@ -1,7 +1,9 @@
 #generate-core-pointer-events-classes (@suffix: ~'') {
 	.pointer-events-none@{suffix}                           {pointer-events: none;}
 	.pointer-events-all@{suffix}                            {pointer-events: all;}
+	.pointer-events-auto@{suffix}                           {pointer-events: auto;}
 
 	.pointer-events-none@{suffix}-when-selected.is-selected {pointer-events: none;}
 	.pointer-events-alld@{suffix}-when-selecte.is-selected  {pointer-events: all;}
```
Comment (nitriques) [inline workspace/assets/css/lib/class-pointer-events.less]: still missing a `d` no ?

Comment (nitriques) [reply workspace/assets/css/lib/class-pointer-events.less]: and a rebase

File: workspace/assets/css/lib/class-pointer-events.less (hunk 1 above)
Comment (PascalPiche) [reply workspace/assets/css/lib/class-pointer-events.less]: Fixed

**Pull request merged**